emBODY/eBcode/arch-arm/board/amcbldc/application07/src/embot_app_application_theMBDagent.cpp

```diff
@@ -392,7 +411,22 @@ void embot::app::application::theMBDagent::Impl::onCurrents_FOC_innerloop(void *
     {
         return;
     }    
-        
+    
+    // performance measurement
+    if(true != useDUMMYforFOC) 
```

Comment:
```suggestion
    if(false == useDUMMYforFOC) 
```

Just to equalize the condition and make it the same as the one above